apps/api/src/workspace-role/workspace-role.service.ts

```diff
@@ -487,14 +512,29 @@ export class WorkspaceRoleService {
     }
 
     for (const slug of slugs) {
-      const project =
-        await this.authorizationService.authorizeUserAccessToProject({
-          slug,
-          authorities: [Authority.READ_PROJECT],
-          user
+      try {
+        // resolve project slug from DB
+        // Read directly from DB to avoid stale cache entries
+        const project = await this.prisma.project.findUnique({

```

Comment:
Revert this change. The only reason we do it like this, is to ensure that the user indeed has access to the projects.